raiden_libs/test/fixtures/client.py

```diff
@@ -0,0 +1,51 @@
+import pytest
+from eth_utils import denoms
+from raiden_libs.test.mocks.client import MockRaidenNode
+
+
+@pytest.fixture
```

Comment:
Doesn't this need session scope in order to work?

Reply:
no, it uses non-session scoped fixtures only.

Reply:
But won't this return a new empty dict every time it's called?

Reply:
Why was this merged without an answer to this question? What is this fixture supposed to do? Is it a WIP?

Reply:
we've clarified this in PM with Paul - the fixture makes using mock client in tests much easier (opening channel is done for both sides automatically)

Reply:
Please keep everything in Github. Not the entire PM conversation but just the result so that your colleagues can follow the logic. 

So from looking at it, it's a test-scope fixture and just functions as a dict that is populated at the `generate_raiden_client` fixture for as many mock clients as you make.